src/main/resources/templates/fragments/common.html

```diff
@@ -92,7 +92,7 @@
           const downloadCompleteText = /*[[#{downloadComplete}]]*/ 'Download Complete';
           window.downloadCompleteText = downloadCompleteText;
           // Create the 'show-game-btn' button
-          var gameButton = $('<button type="button" class="btn btn-primary" id="show-game-btn" style="display:none;">' + boredWaitingText + '</button><br><br>');
```

Comment:
Does this change look alright on the other pages? have you tested?

Reply:
Yes, there where a <br> to much on all forms bevor the Submit Btn.

Example:
Before:
![2024-09-08 23_14_17-Stirling PDF - Auto Split PDF](https://github.com/user-attachments/assets/dcea7cbc-9309-40cb-a0bb-ee73253d8274)

After:
![2024-09-08 23_14_25-Stirling PDF - Auto Split PDF](https://github.com/user-attachments/assets/19a61ad2-6ea3-4e8b-a447-31dd047d9423)


There are still layout issues, but they are not related to the gameButton

Reply:
Perfect, I see there are merge issues
After that and ensuring the change doesn't effect anything we can merge!
Thanks again!